Update current release

EMaksy opened the pull request:
# Description
This pr updates the released documentation in the latest branch.  I was following our our critical hotfix workflow https://www.trento-project.io/docs/developer/trento-docs-site/collaboration-docs-team/docs-git-workflow.html#_3_applying_a_critical_hotfix. 
With this we have also a nice history in the commits :)
<img width="1698" height="654" alt="image" src="https://github.com/user-attachments/assets/dd488928-ad14-40ee-b609-0a44f8202ef5" />

When we release Trento 3.0 we will follow this workflow 
https://www.trento-project.io/docs/developer/trento-docs-site/collaboration-docs-team/docs-git-workflow.html#_2_publishing_a_new_release


## Features

- Added mcp docs images
- Remove docker from user facing docs
- Move updated docker mentions to dev docs
- Update Trento support Plugin
- Add docker deployment

EMaksy commented on content/internal-notes/trento-container-install.adoc:
```diff
@@ -1,19 +1,18 @@
 
-Follow the steps in <<sec-systemd-deployment>>, but skip the *Install
-Trento using RPM packages* step and follow the procedures as described below.
+Follow the steps in link:https://www.trento-project.io/docs/user-guide/trento-install-server.html#sec-systemd-deployment[systemd installation], but skip the *Install
```
Added a link, to not fight with xref

EMaksy commented on content/trento-docs-site/preview-docs-with-docker.adoc:
```diff
@@ -0,0 +1,36 @@
+== Local Documentation Preview with Docker
```
This part is just developer docs for new way of preview antora with docker

EMaksy commented on trento/adoc/trento-sso-integration.adoc:
```diff
@@ -93,44 +93,6 @@ OIDC_CALLBACK_URL=<OIDC_CALLBACK_URL>
 +
 . Restart the application.
 
-==== Enabling OpenID Connect when using Docker images
```
Part of docker mention removal

EMaksy commented on trento/adoc/trento-sso-integration.adoc:
```diff
@@ -496,60 +416,6 @@ SAML_SIGNED_ENVELOPES=<SAML_SIGNED_ENVELOPES>
 +
 . Restart the application.
 
-==== Enabling SAML when using Docker images
```
Part of docker mention removal

dmpop commented on trento/adoc/trento-kubernetes-install.adoc:
```diff
@@ -0,0 +1,267 @@
+[[sec-trento-install-trentoserver-on-existing-k8s-cluster]]
+==== Installing {trserver} on an existing {k8s} cluster
+
+{trserver} consists of a several components delivered as container images and intended for deployment on a {k8s} cluster. A manual production-ready deployment of these components requires {k8s} knowledge. Customers without in-house {k8s} expertise and who want to try {trento} with a minimum of effort, can use the {trento} Helm chart. This approach automates the deployment of all the required comp ...
```
```suggestion
{trserver} consists of a several components delivered as container images and intended for deployment on a {k8s} cluster. A manual production-ready deployment of these components requires {k8s} knowledge. Customers without in-house {k8s} expertise and those who want to try {trento} with a minimum of effort, can use the {trento} Helm chart. This approach automates the deployment of all the required components on a single {k8s} cluster node. You can use the {trento} Helm chart to install {trserver} on a existing {k8s} cluster as follows:
```

dmpop commented on trento/adoc/trento-kubernetes-install.adoc:
```diff
@@ -0,0 +1,267 @@
+[[sec-trento-install-trentoserver-on-k3s]]
+==== Installing {trserver} on K3s
+
+If you do not have a {k8s} cluster, or have one but do not want to use it for {trento}, {suse} Rancher's K3s provides an alternative. To deploy {trserver} on K3s, you need a small server or VM (see <<sec-trento-server-requirements>> for minimum requirements) and follow steps in <<pro-trento-manually-installing>>.
```
```suggestion
If you do not have a {k8s} cluster, or you have one but you do not want to use it for {trento}, you can use {suse} Rancher's K3s as an alternative. To deploy {trserver} on K3s, you need a server or VM (see <<sec-trento-server-requirements>> for minimum requirements) and follow steps in <<pro-trento-manually-installing>>.
```

abravosuse commented on trento/adoc/trento-report-issue.adoc:
```diff
@@ -2,24 +2,23 @@ include::product-attributes.adoc[]
+In case of a systemd deployment, provide the output of the {trento} support plugin, as explained in section <<sec-trento-problemanalysis>>.
 
 In case of a containerized deployment, provide the logs of the
 trento-web and trento-wanda containers. Use `+docker ps+` to retrieve
```
Delete, also, lines 25 and 26. I couldn't not include them in my suggestion.

abravosuse commented on trento/adoc/trento-requirements.adoc:
```diff
@@ -14,7 +14,7 @@ Running all the {trserver} components requires a minimum of 4 GB of RAM, two CPU
-{trserver} supports different deployment scenarios: {k8s}, systemd, and containers. A {k8s}-based deployment of {trserver} is cloud-native and OS-agnostic. It can be performed on the following services:
+{trserver} supports different deployment scenarios: {k8s} and systemd. A {k8s}-based deployment of {trserver} is cloud-native and OS-agnostic. It can be performed on the following services:
 
 * RKE1 ({rancher_k8s_engine} version 1)
```
```suggestion
```

abravosuse reviewed (changes requested):
Great job overall @EMaksy. Just some minor changes. 

If you see a suggestion that changes nothing, it's because I first made a suggestion only valid for the new version. And when I realized it, I undid my changes. Sorry.

EMaksy commented on trento/adoc/trento-requirements.adoc:
```diff
@@ -56,5 +56,5 @@ Similarly, the clusters must have unique authkeys in order to be registered in {
-*  *{trserver}* For a {k8s}-based deployment, you must have access to {suse} public registry for the deployment of {trserver} containers. For a systemd deployment, you must have a registered {sles4sap} 15 (SP3 or higher) distribution. The same applies to a containerized deployment.
-* *{tragent}s* A registered {sles4sap} 15 (SP3 or higher) distribution.
\ No newline at end of file
+* *{trserver}* For a {k8s}-based deployment, you must have access to {suse} public registry for the deployment of {trserver} containers. For a systemd deployment, you must have a registered {sles4sap} 15 (SP4 or higher) or {sles4sap} 16.0 distribution.
```
Maybe skip this change, as the release is soon ?

EMaksy replied:
Done https://github.com/trento-project/docs/pull/121/commits/e64c5a0cafbf0a74e9dd2b0d1f8e62a4b065dcca 
I will create a ticket not to forget to update this back :)

dmpop reviewed (approved):
LGTM

EMaksy commented:
Hey @abravosuse  here is the addressed comments which should be merged with main first, i wrote all the changes in the jira ticket down and will restore them after we update the 2.5 docs
https://github.com/trento-project/docs/pull/122

abravosuse commented:
> Hey @abravosuse here is the addressed comments which should be merged with main first, i wrote all the changes in the jira ticket down and will restore them after we update the 2.5 docs #122

Reviewed and approved! Thanks!

abravosuse commented on trento/adoc/trento-lifecycle.adoc:
```diff
@@ -12,7 +12,7 @@ include::product-attributes.adoc[]
 Delivery mechanism::: RPM package for {sles4sap} 15 SP3 and newer.
 
-Supported runtime::: Supported in {sles4sap} 15 SP3 and newer on x86_64 and ppc64le architectures.
+Supported runtime::: Supported in {sles4sap} 15 SP4 and newer, on x86_64 and ppc64le architectures.
```
```suggestion
Supported runtime::: Supported in {sles4sap} 15 SP3 and newer, on x86_64 and ppc64le architectures.
```

EMaksy replied:
Added fix on main
